lib/rules/cli/4.1.yaml

```diff
@@ -147,6 +147,8 @@
     :f: -f <value>
     :filename: --filename=<value>
     :record: --record=<value>
+    :resource_type: <resource_type>
+    :resouce_name: <resouce_name>
```

Comment:
1. Still typos
2. Where are the parameters used ?

Reply:
@liangxia 
1. Updated.
2. In my Open PR: https://github.com/openshift/cucushift/pull/7303/files